labeler-module/pkg/labeler/labeler_test.go

```diff
@@ -553,7 +553,7 @@ func TestLabeler_handlePodEvent(t *testing.T) {
 				require.NoError(t, err, "failed to update pod status")
 			}
 
-			labeler, err := NewLabeler(cli, time.Minute, "nvidia-dcgm", "nvidia-driver-daemonset")
+			labeler, err := NewLabeler(cli, time.Minute, "nvidia-dcgm", "nvidia-driver-daemonset", "")
```

Comment:
Can we add a test that tests the overrides?

Reply:
yeah, maybe something that does the full flow, i.e., create a node, run the informer and then check the labels. Similar to what we have for the other labels.